airflow/providers/google/cloud/operators/dataproc.py

```diff
@@ -63,6 +73,13 @@
     from airflow.utils.context import Context
 
 
+class PreemptibilityType(Enum):
+    """Contains possible Type values of Preemptibility applicable for every secondary worker of Cluster."""
+
+    PREEMPTIBLE = "PREEMPTIBLE"
+    SPOT = "SPOT"
```

Comment:
Why is `PREEMPTIBILITY_UNSPECIFIED` not an acceptable value?

Reply:
Yes, this value is also acceptable for secondary workers. Good idea to add, thanks!